2020F-usfca-btec-640.github.io/_posts/2020-11-05-welcome-to-jekyll-demo-post.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "Welcome to Jekyll!"
+date:   2020-10-19 12:54:04 -0700
+categories: jekyll update
+---
+You’ll find this post in your `_posts` directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run `jekyll serve`, which launches a web server and auto-regenerates your site when a file is updated.
```

Comment:
This file should not be a part of the PR. Please remove it so we don't have to deal with merge conflicts.